docs/05-technical-reference/00-custom-resources/ac-01-application.md

```diff
@@ -66,4 +66,4 @@ These components use this CR:
 |-----------|-------------|
 | Application Gateway | Reads the API metadata in order to connect to the external system. | 
 | Application Connectivity Validator (in the [Compass mode](../../01-overview/application-connectivity/README.md)) | Validates requests and events from the external system against the respective Application CR. |
-| Runtime Agent (in the [Compass mode](../../01-overview/application-connectivity/README.md)) | Saves the metadata of the connected external system in the Application CR, synchronizes the metadata stored in Compass with the state in the cluster stored in the Application CR. |
+| Runtime Agent (in the [Compass mode](../../01-overview/application-connectivity/README.md)) | Saves the metadata of the connected external system in the Application CR, synchronizes the metadata stored in Compass with the state in the cluster stored in the Application CR. |
```

Comment:
So the Application CR is still part of Kyma and is not supposed to be removed from the documentation?

Reply:
Yes, as [Przemek suggested](https://github.com/kyma-project/kyma/pull/18382#discussion_r1385125054)